Sources/LocalReceiptParsing/DataConverters/UInt8+Extensions.swift

```diff
@@ -68,18 +68,8 @@ private extension UInt8 {
     /// - Throws: `BitShiftError`
     func maskForRange(_ range: UInt8) throws -> UInt8 {
         guard 0 <= range && range <= 8 else { throw BitShiftError.rangeLargerThanByte }
-        switch range {
-        case 1: return 0b1
-        case 2: return 0b11
-        case 3: return 0b111
-        case 4: return 0b1111
-        case 5: return 0b11111
-        case 6: return 0b111111
-        case 7: return 0b1111111
-        case 8: return 0b11111111
-        default:
-            throw BitShiftError.unhandledRange
-        }
+
+        return 0b11111111 >> (8 - range)
```

Comment:
One liner, but also avoids jumps. I don't think the optimizer could have figured this out.

Reply:
I like it, but it might be worth it to add a couple examples for clarification?

```
// If range 1, return `0b1`
// If range 2, return `0b11`
// ...
```

Reply:
Done.